webrender/src/resource_cache.rs

```diff
@@ -174,7 +167,7 @@ where
             .cloned()
             .collect::<Vec<_>>();
         for key in resources_to_destroy {
-            self.resources.remove(&key);
+            self.resources.remove(&key).unwrap();
```

Comment:
I revert the change, but the CI test is failed.

error: unused `std::result::Result` which must be used
   --> src/resource_cache.rs:170:13
    |
170 |             self.resources.remove(&key).unwrap();
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D unused-must-use` implied by `-D warnings`